liblttng-ust-java-agent/java/lttng-ust-agent-jul/org/lttng/ust/agent/jul/LttngLogHandler.java

```diff
@@ -84,11 +88,30 @@ public LttngLogHandler() throws IOException, SecurityException {
 			throw new IOException(e);
 		}
 
+		/* Initialize handler specific properties */
+		LogManager manager = LogManager.getLogManager();
+		String cname = getClass().getName();
+		logSourceMethod = (getBooleanProperty(manager.getProperty(cname + ".logSourceMethod"), true));
+		logSourceClass = (getBooleanProperty(manager.getProperty(cname + ".logSourceClass"), true));
+
 		/** Register to the relevant agent */
 		agent = LttngJulAgent.getInstance();
 		agent.registerHandler(this);
 	}
 
+	private boolean getBooleanProperty(String val, boolean defaultValue) {
+	        if (val == null) {
```

Comment:
grrr. I was using the Eclipse (Trace Compass) default format. I'll have to set the format for this project...